WinImageBuilder.psm1

```diff
@@ -582,15 +574,6 @@ function Create-VirtualSwitch
     return $sw
 }
 
-function Check-Prerequisites
-{
-    $needsHyperV = Get-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V
-    if ($needsHyperV.State -ne "Enabled"){
-        Write-Warning $noHypervWarning
-        exit 1
-    }
-}
-
 function GetOrCreateSwitch
 {
     $vmSwitches = Get-VMSwitch -SwitchType external

```

Comment:
From what I know this is a Hyper-V cmdlet so you have to check if the role is enabled before doing this... It's pointless anyway (even if by miracle it would work) to do this check in any other case.